Caboodle/Clipboard/Clipboard.android.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Android.Content;
+using Android.Runtime;
+using Android.Views;
+using System.Threading.Tasks;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Clipboard
+    {
+        public static void SetText(string text)
+        {
+            var clipboardManager = (ClipboardManager)Application.Context.GetSystemService(Context.ClipboardService);
```

Comment:
Should make this a static helper such as:
https://github.com/xamarin/Caboodle/blob/connectivity/Caboodle/Connectivity/Connectivity.android.cs#L17-L26